modules/ossm-integrating-with-user-workload-monitoring.adoc

```diff
@@ -149,10 +199,11 @@ spec:
       action: replace
       targetLabel: pod_name
     - action: replace
-      replacement: "<smcp_name>-<smcp_namespace>" # <2>
-      targetLabel: mesh_id
+      replacement: "basic-istio-system" # <2>
+      targetLabel: mesh_id # <3>
 ----
 <1> Since {product-title} monitoring ignores the `namespaceSelector` spec in `ServiceMonitor` and `PodMonitor` objects, you must apply the `PodMonitor` object in all mesh namespaces, including the control plane namespace.
-<2> A shared Prometheus instance can include metrics from multiple meshes. However, if you are deploying Kiali, the metrics must be associated with a single mesh. To associate metrics with a single mesh, add a unique `mesh_id` to each one. Doing so narrows the query scope in Kiali to only the relevant mesh metrics.
+<2> Replace `"basic-istio-system"` with a unique value, such as `<smcp-name>-<smcp-namespace>` to create a unique combination within a cluster with multiple SMCPs. 
```

Comment:
```suggestion
<2> `"basic-istio-system"` is a combination of the SMCP name and namespace. It can be replaced with any other unique value within a cluster with multiple SMCPs. This is required only when Kiali is used and multiple SMCPs are deployed in a cluster. In case of cluster-wide deployment, it can be skipped.
```